client/wc-api/onboarding/operations.js

```diff
@@ -170,6 +171,28 @@ function activatePluginToResource( response, items ) {
 	return resources;
 }
 
+function readJetpackStatus( resourceNames, fetch ) {
+	const resourceName = 'jetpack-status';
+
+	if ( resourceNames.includes( resourceName ) ) {
+		const url = JETPACK_NAMESPACE + '/connection';
+
+		return [
+			fetch( {
+				path: url,
+			} )
+				.then( response => {
+					return { [ resourceName ]: { data: response } };
+				} )
+				.catch( error => {
+					return { [ resourceName ]: { error: String( error.message ) } };
```

Comment:
Do you think we should return the full error here or just the string?  I've been returning the full error "just in case" we need it, but might be better if I update mine to just the string to keep things consistent if there aren't any use cases for the error codes.